src/modules/core/components/Tag/Tag.css

```diff
@@ -46,6 +46,12 @@
   composes: baseStyles;
 }
 
+.themeBanned {
+  composes: baseStyles;
+  background-color: var(--banned);
```

Comment:
as mentioned in my comment for the variables file, it should use color-mod and --var-pink directly here

Reply:
Fixed on next push!